### ISSUE TITLE
Side Navigation: Does not support the `prefers-reduced-motion` CSS media query

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2355]
The side navigation component is quite animation heavy and could cause issues for some users.

We should have minimal motion if a user has `prefers-reduced-motion` accessibility option set.
